src/elements/form.ts

```diff
@@ -0,0 +1,31 @@
+import { LitElement, html } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+@customElement('hangman-form')
+export class HangmanForm extends LitElement {
+  inputLetter: string = ''
+
+  render() {
+    return html` <form id="form">
+      <hangman-input
+        @letter-changed="${({ detail }: { detail: string }) => (this.inputLetter = detail)}"
+      ></hangman-input>
+      <hangman-button @click="${this._handleSubmit}">Guess</hangman-button>
+    </form>`
+  }
+  @property()
+  private _handleSubmit() {
+    if (!this.inputLetter) return
+    this.dispatchEvent(
+      new CustomEvent('letter-changed', {
```

Comment:
how do you feel about renaming this event to for example `submit` or `guess`? I think `letter-changed` sounds like i would expect it to trigger every time the user types a letter into the input.

Reply:
Guess sounds good. It also gives more of the 'game' feeling.